sdk/keyvault/Azure.Security.KeyVault.Keys/src/Azure.Security.KeyVault.Keys.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
what changed here?

Reply:
Honestly, I'm quite sure. I'm assuming the BOM (perhaps removal of one), but even looking at `git diff` it wasn't obvious like it always has been. But it's what the `Update-Release.ps1` script did so I didn't want to argue with it (like I already have to because of Azure/azure-sdk-tools#1389).

Reply:
Just FYI, normally the removal of a BOM looks much different. It almost seems like there was a leading space:

![image](https://user-images.githubusercontent.com/1532486/110549013-4e55a080-80e6-11eb-812f-1451376536b2.png)